### ISSUE TITLE
Wrong OS bits size on mingw64 on Windows 10 x64

### ISSUE THREAD

**Issue by artemshein** [rdp/os#32]
Same OS, different Rubies (64-bit and 32-bit):
$ uname -a
MINGW64_NT-10.0 name 2.5.0(0.295/5/3) 2016-03-31 18:47 x86_64 Msys
$ ruby -e 'require "os";p RUBY_PLATFORM;p OS.bits'
"x64-mingw32"
64
$ ruby -e 'require "os";p RUBY_PLATFORM;p OS.bits'
"i386-mingw32"
32

**Comment by artemshein** [comment]
The only solution I found is:
```
def is_64bit_win?
  ENV.has_key?('ProgramFiles(x86)')
end
```
Ugly, but works

**Comment by rdp** [comment]
Yeah that's a tricky one, since it's basically "os differs from runtime"
and I never did figure out an awesome solution there.  Maybe another method
"OS.os_bits" LOL?

**Comment by rdp** [comment]
pr welcome!
